templates/ciena_6x_vlan_show.textfsm

```diff
@@ -0,0 +1,10 @@
+Value VLAN_ID (\d+)
+Value VLAN_NAME ([0-9a-zA-Z_#]+)
```

Comment:
```suggestion
Value NAME ([0-9a-zA-Z_#]+)
```

This will match `cisco_ios_show_vlan.textfsm` value names.

Reply:
I've made the change in my template and will push that with NAME.